### ISSUE TITLE
Add type annotations?

### ISSUE THREAD

**Issue by benbariteau** [zeek/pysubnettree#27]
[Type annotations](https://docs.python.org/3/library/typing.html) are a useful feature for consumers of any library, so it would be nice if this library had it as well.

**Comment by rsmmr** [comment]
Mind providing more context?

**Comment by benbariteau** [comment]
I edited it, does that make sense?

**Comment by rsmmr** [comment]
Yeah, thanks, would be nice to have. Contributions welcome on this.
